packages/components/src/tripInput/__tests__/TripInput.test.js

```diff
@@ -12,7 +12,10 @@ it('renders', () => {
     shallow(
       <TripInput
         onPress={jest.fn()}
-        icon={<IconMock />}
+        icon={
+          // $FlowFixMe Expected Error
```

Comment:
Could you use `$FlowExpectedError: ` with a comment of why you expect the error?